x-pack/plugins/translations/translations/ja-JP.json

```diff
@@ -1669,12 +1669,12 @@
     "data.functions.esaggs.help": "AggConfig 集約を実行します",
     "data.functions.esaggs.inspector.dataRequest.description": "このリクエストはElasticsearchにクエリし、ビジュアライゼーション用のデータを取得します。",
     "data.functions.esaggs.inspector.dataRequest.title": "データ",
-    "data.functions.indexPatternLoad.help": "インデックスパターンを読み込みます",
-    "data.functions.indexPatternLoad.id.help": "読み込むインデックスパターンID",
-    "data.indexPatterns.ensureDefaultIndexPattern.bannerLabel": "Kibanaでデータの可視化と閲覧を行うには、Elasticsearchからデータを取得するためのインデックスパターンの作成が必要です。",
-    "data.indexPatterns.fetchFieldErrorTitle": "インデックスパターンのフィールド取得中にエラーが発生 {title}（ID：{id}）",
-    "data.indexPatterns.indexPatternLoad.error.kibanaRequest": "サーバーでこの検索を実行するには、KibanaRequest が必要です。式実行パラメーターに要求オブジェクトを渡してください。",
-    "data.indexPatterns.unableWriteLabel": "インデックスパターンを書き込めません。このインデックスパターンへの最新の変更を取得するには、ページを更新してください。",
+    "dataViews.indexPatternLoad.help": "インデックスパターンを読み込みます",
```

Comment:
I think need to also change sorting and move it into the right place. otherwise will trigger changes for someone else during regeneration

Reply:
These don't appear to be alphabetical to me, at least there are pretty significant exceptions.